src/Controllers/Api/ShiftsController.php

```diff
@@ -143,7 +143,7 @@ protected function shiftEntriesResponse(Collection $shifts): Response
 
                 $entries = $entries->map(fn(ShiftEntry $entry) => [
                     'user' => UserResource::toIdentifierArray($entry->user),
-                    'freeloaded' => $entry->freeloaded,
+                    'freeloaded' => $entry->freeloaded_by,
```

Comment:
```suggestion
                    'freeloaded' => $entry->freeloaded_by ? UserResource::toIdentifierArray($entry->freeloaded_by) : null,
```

This change also impacts the API definition which must be changed accordingly (and using the `User Resource` is also needed for consistency)

Reply:
<img width="925" alt="Screenshot 2024-11-11 at 15 45 01" src="https://github.com/user-attachments/assets/a78f7bda-96a5-4756-9f4e-4c1fe5b640b4">
my IDE does not like this